platform/qt/src/qmapboxgl.cpp

```diff
@@ -76,6 +76,7 @@ static_assert(mbgl::underlying_type(QMapboxGL::MapChangeDidFinishRenderingFrameF
 static_assert(mbgl::underlying_type(QMapboxGL::MapChangeWillStartRenderingMap) == mbgl::underlying_type(mbgl::MapChangeWillStartRenderingMap), "error");
 static_assert(mbgl::underlying_type(QMapboxGL::MapChangeDidFinishRenderingMap) == mbgl::underlying_type(mbgl::MapChangeDidFinishRenderingMap), "error");
 static_assert(mbgl::underlying_type(QMapboxGL::MapChangeDidFinishRenderingMapFullyRendered) == mbgl::underlying_type(mbgl::MapChangeDidFinishRenderingMapFullyRendered), "error");
+static_assert(mbgl::underlying_type(QMapboxGL::MapChangeSourceAttributionDidChange) == mbgl::underlying_type(mbgl::MapChangeSourceAttributionDidChange), "error");
```

Comment:
> Do I need to add it anywhere else?

That is not it, this code [won't compile on Qt](https://travis-ci.org/mapbox/mapbox-gl-native/jobs/161966172#L1279) because we are missing the `MapChangeDidFinishLoadingStyle` signal on the API that must be declared before `MapChangeSourceAttributionDidChange` otherwise the enum won't fully match the counterpart which is exactly what the `static_assert` is asserting.